src/commoncode/fileutils.py

```diff
@@ -51,7 +51,10 @@
 import stat
 import sys
 import tempfile
-
+from functools import partial
+from hashlib import md5
+from os.path import abspath
+from os.path import dirname
```

Comment:
Do you mind keeping the imports sorted in a given "block" (e.g. here the stdlib imports)